docs/89-NEWS.md

```diff
@@ -1,10 +1,28 @@
 # NEWS
 
-<a name="v43"></a>
-# Awesome window manager framework version 4.3 changes
+<a name="v44"></a>
+# Awesome window manager framework version 4.4 changes
```

Comment:
It would be nice to add the word "draft" or avoid using 4.4. This page is published on the website and the git doc is used by many because it is always better than the stable one. Having a release note for an unreleased version might be confusing.

Beside, lets merge this.